<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>802</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Программа</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>801</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>9</y>
       <width>771</width>
       <height>621</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Положение объекта</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>761</width>
         <height>591</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="objectLocationLat" native="true">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>13</y>
          <width>735</width>
          <height>278</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="objectLocationLong" native="true">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>301</y>
          <width>735</width>
          <height>277</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Наблюдаемые спутники</string>
      </attribute>
      <widget class="QWidget" name="currSats" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>751</width>
         <height>571</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Выбранный спутник</string>
      </attribute>
      <widget class="QWidget" name="chosenSat" native="true">
       <property name="geometry">
        <rect>
         <x>386</x>
         <y>306</y>
         <width>364</width>
         <height>274</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="sputnik_Z" native="true">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>306</y>
         <width>365</width>
         <height>274</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="sputnik_X" native="true">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>21</y>
         <width>365</width>
         <height>275</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="sputnik_Y" native="true">
       <property name="geometry">
        <rect>
         <x>386</x>
         <y>21</y>
         <width>364</width>
         <height>275</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>280</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Системы ГНСС</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>291</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>GPS</string>
      </attribute>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>12</y>
         <width>155</width>
         <height>230</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>GLONASS</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QListView" name="listView_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="tableViewObjectInfo">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>60</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>10</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QLabel" name="simulationTimer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tableViewLabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Информация об объекте</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл для парсинга</string>
    </property>
    <addaction name="open_file"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_file">
   <property name="text">
    <string>Выбрать файл для парсинга</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
